File: lessons/5.2/1-test.js
const MongoClient = require('mongodb').MongoClient;
const assert = require('assert');

const uri = 'mongodb://localhost:27017';
const db = 'cooker';

console.log("Setting up mongo client");

const client = new MongoClient(uri, { useUnifiedTopology: true});

console.log("Connecting to database ...")

client.connect(function(err) {
  assert.equal(null, err);

  console.log("Connected Successfully!");

  const database = client.db(db);

  console.log("The database name is: " + database.databaseName);

  client.close();
});

File: lessons/5.1/1-test.py
#! /usr/bin/python

import pymongo

print("Setting up mongo client")

client = pymongo.MongoClient("localhost", 27017)

print("Connecting to database ...")

# set our database to "cooker"
db = client.cooker

database_name = db.name

print("Connected Successfully!")

# output DB name
print("The database name is: {}".format(database_name))

